.gitignore/src/main/java/com/task/model/User.java

```diff
@@ -0,0 +1,60 @@
+package com.task.model;
+
+import lombok.Data;
+import javax.persistence.*;
+
+@Data
+@Table(name = "user")
+@Entity
+public class User // implements UserDetails {
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "ID")
+    private Long id;
+    @Column(name = "USERID")
+    private String userId;
+    @Column(name = "PROFILENAME")
+    private String profileName;
+    @Column(name = "role")
+    private String role;
+    @Column(name = "password")
+    private String password;
+/*
+    @Override
+    public Collection<? extends GrantedAuthority> getAuthorities() {

```

Comment:
please, don't leave commented code or implement it. as an option, you can create a // TODO comment with explanation of future task... but don't leave commented code